modules/sony/display/cbx.rb

```diff
@@ -0,0 +1,61 @@
+module Sony; end
+module Sony::Display; end
+
+# Documentation: https://docs.google.com/spreadsheets/d/1F8RyaDqLYlKBT7fHJ_PT7yHwH3vVMZue4zUc5McJVYM/edit?usp=sharing
+
+class Sony::Display::CBX
+    include ::Orchestrator::Constants
+    include ::Orchestrator::Transcoder
+
+    # Discovery Information
+    tcp_port 4999
+    descriptive_name 'Sony CBX Display'
+    generic_name :Display
+
+    def on_load
+        on_update
+    end
+
+    def on_update
+    end
+
+    def connected
+        schedule.every('60s') { power? }
+    end
+
+    def disconnected
+        schedule.clear
+    end
+
+    #
+    # Power commands
+    #
+    def power(state, opt = nil)
+        if is_affirmative?(state)
+            send("8C 00 00 02 01 8F", hex_string: true)
+            logger.debug "-- sony display requested to power on"
+        else
+            send("8C 00 00 02 01 8F", hex_string: true)
+            logger.debug "-- sony display requested to power off"
+        end
+
+        # Request status update
+        power?
```

Comment:
Does the device provide a direct response to the power command. If so, it would be preferable to parse this directly and remove the follow up query.

Reply:
nope...